index.html

```diff
@@ -294,9 +294,42 @@ <h1> Skils</h1>
             </div>
         </section>
         <!-- end of about -->
-        <!-- <section  class="get-in-touch">
-            <h1>Get in</h1>
-        </section> -->
+        <section  class="get-in-touch">
+            <h3 hidden>heading</h3>
+            <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+            </p>
+            <form action="/" method ="post">
+                <div class="input-control">
+                    <input class="fullname-control" type="text" id="full-name" placeholder="Full name" required>
```

Comment:
- [ ] Name input (can be first name and last name field depending on the template used) has `maxlength="30"` attribute. Let's add `maxlength` here.

Reply:
I am following a figma design  that i chosen not the picture on the requirements, as the design differ from one other, for my design no  first name and last name fields